packages/ember-runtime/lib/system/native_array.js

```diff
@@ -138,9 +138,10 @@ var A;
 
 if (Ember.EXTEND_PROTOTYPES === true || Ember.EXTEND_PROTOTYPES.Array) {
   NativeArray.apply(Array.prototype);
-  A = function (arr = []) { return arr; };
+  A = function (arr = []) { return arr || []; };
 } else {
   A = function (arr = []) {
+    if (arr === null) { arr = []; }
```

Comment:
Dunno- I would defer to the old behavior since this is a regression fix. Can you perhaps try it out on 2.4 or and older version and simply match that? Thanks man.